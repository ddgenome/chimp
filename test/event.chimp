# test the event based rate constant conversion
mechanism "event.mech"
parameter "event.coverage.par"
task "event.coverage.task"
parameter "event.event.par"
task "event.event.task"
