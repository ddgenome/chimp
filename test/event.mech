# mechanism to test event based rate constant conversion
A + @ -> k_constant(k_Aads) <- k_arrhenius(A_Ades, E_Ades) @A;
@A -> k_arrhenius(A_sr, E_sr) <- k_arrhenius(A_Bads, E_Bads) B + @;
