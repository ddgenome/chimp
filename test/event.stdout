chimp 0.1
Copyright (C) 2004 David Dooling <banjo@users.sourceforge.net>

There is ABSOLUTELY NO WARRANTY; see file COPYING for details.
This is free software, and you are welcome to redistribute it
under certain conditions; see file COPYING for details.

Report bugs to http://sourceforge.net/projects/chimp/.
chimp: performing task event.coverage.par... completed; 0u 0s 0w
chimp: performing task event_coverage... completed; 0.74u 0.01s 0.75w
chimp: performing task event.event.par... completed; 0u 0s 0w
chimp: performing task event_event... completed; 0.74u 0s 0.74w
